idlpy/src/types.c

```diff
@@ -175,6 +216,9 @@ emit_field(
     }
 
     if (idl_is_member(parent)) {
+
+        //printf("emit_field idl_is_member name : %s , type %s \n", name , type);
```

Comment:
Please don't leave commented code in the PR.

Reply:
all the printfs that we have added have been deleted